.circleci/config.yml

```diff
@@ -312,24 +312,16 @@ jobs:
     executor: trivy_executor
     steps:
       - prepare
-      - setup_remote_docker:
-          docker_layer_caching: false
       - run:
           name: Install trivy
-          command: |
-            apk add --update-cache --upgrade curl bash
-            curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin
-      # remove the test files and only use the folder names
+          command: curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b $HOME/bin
+      # scan both develop-arm64 and develop-amd64 images using remote src.
       - run:
           name: Scan with trivy
           shell: /bin/sh
           command: |
-            rm -rf docker/test*
-            for FILE in $(ls docker)
```

Comment:
What were the previous values of $FILE?

Reply:
we used to have `docker/jdk17/Dockerfile` and `docker/jdk11/Dockerfile`. Hence we used to create/publish `consensys/web3signer:develop-java11` and `consensys/web3signer:develop-java17`. Since we now default to java17 variant, the `develop` tag now points to `develop-java17`. In addition, we further publish platform specific variants as well i.e. `develop-amd64` and `develop-arm64`. Similarly, `develop-java17-amd64` and `develop-java17-arm64` 

In summary, we only need to test/scan `develop` tag unless we introduce `java21` tag along with `java17` tag. We were also only scanning `develop-amd64` variant in past as CircleCI trivy job runs on Linux amd64 environment. Now we make sure to scan both amd64 and arm64 variants (as both of them are built separately).